freex/.gitignore

```
@@ -22,3 +22,4 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+firebase.config
```

```suggestion
*firebase.config
*firebase.config.js
```

Plik po prostu usuńcie. Po wprowadzeniu tej zmiany nie powinien się pojawić w commicie ponownie.